<?xml version="1.0" ?>
<launch>
   <arg name="token" default="target"/>
   <arg name="timeout" default="5.0"/>

   <!-- Start the node -->
   <node name="target_node" pkg="target_estimation" type="target_node" respawn="false" output="screen">
        <param name="token" value="$(arg token)"/>
        <param name="timeout" value="$(arg timeout)"/>
   </node>
</launch>
